### PR TITLE
Add Beachball to Repo #2

### PR DESCRIPTION
**Pull request by pkanher617**
How it works:

1. Developer creates a PR that affects X packages.
2. The "Beachball Check" workflow will check whether the developer has committed changefiles, and will fail and prevent merging if it does not detect them.
3. The developer run "npm run beachball:change" at the root of the repo locally on their PR branch and describes the changes to each affected package. Beachball will generate changefiles and commit them to the branch. The developer should push these changefiles and merge them when the PR is approved.
4. Changefiles are collected on dev. When all the files are ready to be published, a PR to master is created. On merge, the `beachball publish` command is run, which reviews the changefiles for the highest order release (i.e. major, minor, patch), then bumps the package versions and their dependency versions accordingly, and finally performs the publish.

### CODE REVIEW

File: samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "shouldPublish": false
```
Comment (tnorling) [inline samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc:2]: Is there a way to ignore the samples folder completely so we don't have to add this to every sample?

Comment (pkanher617) [reply samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc:2]: I wanted to use the path but beachball is currently overwriting the path config parameter with `cwd` so until they fix their config we need to do this.

Comment (tnorling) [reply samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc:2]: Got it. Makes sense

Comment (pkanher617) [reply samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc:2]: I opened a ticket here: https://github.com/microsoft/beachball/issues/377

File: change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Add beachball config",
+  "packageName": "@azure/msal-angularjs",
+  "email": "prkanher@microsoft.com",
```
Comment (sangonzal) [inline change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json:5]: Should we be using team alias or even nugetaad email instead?

Comment (pkanher617) [reply change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json:5]: This is based off of the github account that made PR, it puts the author next to the item in the changelog. Not sure if we want to change this?

**Comment by pkanher617** [comment]
Example of what is added to the changelog for a minor version change: 

```
## 1.4.0

Fri, 07 Aug 2020 21:42:47 GMT

### Minor changes

- updating files for automated release steps (prkanher@microsoft.com)

```

File: change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json (hunk 2 above)
Comment (sangonzal) [reply change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json:5]: I assumed this was the email alias that was used for releases. Makes sense to have the author who made the change for changelog purposes.

**Pull request merged**